app/src/main/res/values-vi/strings.xml

```diff
@@ -1,8 +1,73 @@
 <!--https://newbedev.com/concatenate-multiple-strings-in-xml-->
 <resources>
     <!-- KEYS DON'T TRANSLATE -->
+    <string name="search_providers_list_key" translatable="false">search_providers_list</string>
```

Comment:
please remove all the entries which have `translatable="false"`, they cant be translated